### ISSUE TITLE
devtools::release() trashes NAMESPACE

### ISSUE THREAD

**Issue by friendly** [hadley/devtools#1437]
I have a legacy package developed without using roxygen for the .Rd files.  When I tried to use `devtools::release()` to submit the package, rogygen2 was run, resulting in an empty NAMESPACE file, with no prompt for 'first time use of roxygen2`.  This seems to be a bug, but certainly unexpected.
Is there any way to prevent this?

    > devtools::release()
    Warning: DR_DEVTOOLS FOUND PROBLEMS
    * R is out of date (3.2.5 vs 3.3.2)
    * RStudio is out of date (0.99.1266 vs 1.0.136)
    Proceed anyway?
    1: Uhhhh... Maybe?
    2: I agree
    3: I forget

    Selection: 2
    Building and checking HistData ===========================================================================================
    Updating HistData documentation
    Loading HistData
    First time using roxygen2. Upgrading automatically...
    Updating roxygen version in  C:\R\projects\histdata\pkg/DESCRIPTION 
    Writing NAMESPACE

**Comment by jimhester** [comment]
The issue is `devtools::release()` calls `devtools::check()` with the default argument `document = TRUE`. You can run the check manually then run `devtools::release()` without the check.

```r
devtools::check(document = FALSE)
devtools::release(check = FALSE)
```

**Comment by levimcclenny** [comment]
I actually had the same issue, when calling devtools::release() my NAMESPACE was overwritten with a blank one, resulting in some random warnings and notes right before submission. Funny part is that I used roxygen2 to create the initial NAMESPACE anyway, so I'm not sure about the comment 'first time use of roxygen2'

I believe this is a bug, but I got my package submitted using `devtools::release(check = FALSE)` regardless. I just ran an extra check before submitting.
